packit/api.py

```diff
@@ -105,8 +105,12 @@ def sync(
     ):
 
         if add_new_sources:
-            archive = distgit.download_upstream_archive()
-            if not distgit.is_archive_on_lookaside_cache(archive):
+            archive_name = distgit.get_upstream_archive_name()
+            # btw this is really naive: the name could be the same but the hash can be different
+            # we should do something when such situation happens
```

Comment:
:+1:

Reply:
okay, let me open an issue for that; Ken Dreyer of ceph raised this originally